flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/TimerService.java

```diff
@@ -35,43 +36,43 @@
 @Internal
 public interface TimerService extends ProcessingTimeService {
 
-    /**
-     * Returns <tt>true</tt> if the service has been shut down, <tt>false</tt> otherwise.
-     */
-    boolean isTerminated();
+	/**
```

Comment:
Nit: formatting changed from first commit. This file should not have any changes in this commit.